pkg/models/spdk.go

```diff
@@ -37,3 +37,11 @@ type NpiQosBwIopsLimitParams struct {
 
 // NpiQosBwIopsLimitResult is the result of setting QoS limits
 type NpiQosBwIopsLimitResult bool
+
+// VhostCreateBlkControllerParams holds parameters to create a virtio-blk device
+type VhostCreateBlkControllerParams struct {
```

Comment:
can we prefix type with `Mev` or `Npi` ?

Reply:
Added `Mev`